_posts/2023-07-03-introducing-palletjack.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Introducing palletjack: Updating AGOL feature services from external sources"
+author:
+  display_name: Jake Adams
+  email: jdadams@utah.gov
+date: 2023-07-03 11:00:00
+categories:
+  - Featured
+  - Developer
+tags:
+  - python
+---
+
+_From the I Love Being Lazy department_
+{: .text-center}
```

Comment:
**thought**: This is what the news preview looks like. 

<img width="416" alt="image" src="https://github.com/agrc/gis.utah.gov/assets/325813/11573e63-1856-457e-a6a7-18d3225f5c33">

Reply:
Yeah, I'm ok with that. Do you have serious heartburn about it?

Reply:
nope. just letting you know.